frontend/public/components/parameters.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
```

Comment:
I might call the file service-catalog-parameters.tsx to make it clear this is only for service catalog since other things have parameters, too